core/src/style/textStyle.cpp

```diff
@@ -97,6 +97,7 @@ void TextStyle::buildPolygon(const Polygon& _polygon, const DrawRule& _rule, con
             n++;
         }
     }
+    if (n == 0) { return; }
```

Comment:
Ideally we should return much before n is calculated... when polygon size is zero.